Cargo.toml

```diff
@@ -39,4 +39,4 @@ debug-assertions = true
 overflow-checks = true
 rpath = false
 lto = "thin"
-incremental = false
+incremental = true
```

Comment:
to speedup linking?

Reply:
sorry, yes. I should not have included this in the PR though.